Q1.java

```diff
@@ -0,0 +1,95 @@
+import java.util.*;
+
+class Q1
+{
+	public static void main(String[] args)
```

Comment:
According to the Google Java style guide and Oracle Java code conventions there should be no line break before the opening brace.